tests/unit/test_config.py

```diff
@@ -473,19 +457,30 @@ def test_process_startcp(
             {
                 'initial cycle point': '2021',
                 'final cycle point': None,
-                'final cycle point constraints': []
             },
             None,
             None,
             None,
             id="No fcp"
         ),
+        pytest.param(
+            ISO8601_CYCLING_TYPE,
+            {
+                'initial cycle point': '2021',
+                'final cycle point': '',
+            },
+            None,
+            None,
+            None,
+            id="Empty fcp in cfg"
+            # Need to check this as fcp treated as string by parsec, unlike
+            # other cycle point settings
```

Comment:
TODO?

Reply:
"Need to check this as" == "This test is needed because"